src/scf-release/src/acceptance-tests-brain/test-scripts/minibroker_helper.rb

```diff
@@ -128,7 +133,14 @@ def run_test
 
             File.open("#{tmpdir}/service-params.json", 'w') { |f| f.puts service_params.to_json }
             run "jq -C . #{tmpdir}/service-params.json"
+            started_service_creation = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
Not sure we need that hard core sort of timing, but sure

Reply:
🤷